types/chai-dom/chai-dom-tests.ts

```diff
@@ -8,7 +8,7 @@ function test() {
 
     var testElement = '<div></div>';
     expect(testElement).to.have.attribute('foo', 'bar');
-    expect(testElement).to.have.attr('foo').match(/bar/);
+    expect(testElement).to.have.attr('foo').match('bar');
```

Comment:
Assertion in chai-dom using `/bar/` in their example. Could you explain why this change to string?

http://chaijs.com/plugins/chai-dom/

Reply:
That seems to be a mistake, as does the other change you pointed out. I thought I had reverted those changes once I figured out what was going on, but it appears I didn't. I agree that those parameters should still be RegExp since they're testing for a string match (the default chai behavior.) I'll amend my previous commits to reflect those fixes. Thanks for pointing this out.